test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -273,11 +275,26 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 		await driverHelper.waitTillNotPresent( this.driver, inserterMenuSelector );
 	}
 
+	/**
+	 * Returns a list of titles for the block items currently shown in the main inserter.
+	 *
+	 * @returns {string[]} Array of block titles (i.e ['Open Table', 'Paypal']);
+	 */
+	async getShownBlockInserterItems() {
+		return this.driver
+			.findElements(
+				By.css(
+					'.edit-post-layout__inserter-panel .block-editor-block-types-list span.block-editor-block-types-list__item-title'
+				)
+			)
+			.then( ( els ) => Promise.all( els.map( ( el ) => el.getAttribute( 'innerText' ) ) ) );
```

Comment:
You can use [`webdriver.promise.map`](https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/lib/promise.html#map) instead. Similar to as we do i.e. with `filter` [here](https://github.com/Automattic/wp-calypso/blob/4b88b39e63812a8673ad4edf6437e51fb3c76169/test%2Fe2e%2Flib%2Fdriver-helper.js#L531).

Reply:
Oh, nice! TIL 👍🏻